### PR TITLE
BZ-2055147: correcting output of oc get bmh

### PR DESCRIPTION
**Pull request by kquinn1204**
Addresses BZ https://bugzilla.redhat.com/show_bug.cgi?id=2055147

Preview: [Preview link](https://deploy-preview-42550--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.html)

Relevant for 4.10 and needs merged back to 4.9

### CODE REVIEW

**Comment by kquinn1204** [comment]
@elfosardo please review the PR that addresses https://bugzilla.redhat.com/show_bug.cgi?id=2055147

File: modules/ipi-install-provisioning-the-bare-metal-node.adoc [hunk 1]
```diff
@@ -74,20 +74,20 @@ Replace `<num>` with the new number of worker nodes. Replace `<machineset>` with
...
 [source,bash]
 ----
-NAME                 STATUS   PROVISIONING STATUS   CONSUMER                  BMC                 HARDWARE PROFILE   ONLINE   ERROR
-openshift-worker-<num>   OK       provisioning          openshift-worker-<num>-65tjz   ipmi://<out-of-band-ip>   unknown            true
+NAME                    STATE             CONSUMER                          ONLINE   ERROR
+openshift-worker-<num>  provisioning      openshift-worker-<num>-65tjz      true
 ----
 +
-The `provisioning` status remains until the {product-title} cluster provisions the node. This can take 30 minutes or more. After the node is provisioned, the status will change to `provisioned`.
+The `provisioning` state remains until the {product-title} cluster provisions the node. This can take 30 minutes or more. After the node is provisioned, the state will change to `provisioned`.
```
Comment (elfosardo) [inline modules/ipi-install-provisioning-the-bare-metal-node.adoc:85]: this should be "The `provisioning status`"

**Review by elfosardo — CHANGES_REQUESTED** [review]

**Review by elfosardo — APPROVED** [review]

**Review by reihl — APPROVED** [review]

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**